share/misc/bsd-family-tree

```diff
@@ -447,39 +447,45 @@ FreeBSD 5.2           |      |                 |                       |
  |  FreeBSD           |      |                    |                    |
  |   13.2             |      |                    |                    |
  |                    |      |                    |                    |
-FreeBSD 14 -current   |  NetBSD -current   OpenBSD -current    DragonFly -current
+ *--FreeBSD           |      |                    |                    |
+ |   14.0 prerelease  |      |                    |                    |
+ |                    |      |                    |                    |
+FreeBSD 15.0-CURRENT  |  NetBSD-current  OpenBSD-current  DragonFly-DEVELOPMENT
```

Comment:
I'd omit the major number here. It causes too much churt. This line should basically never change.

Reply:
Resolved by [`9afd0fd` (#812)](https://github.com/freebsd/freebsd-src/pull/812/commits/9afd0fd2dda7118ef95d8c81edfe92e51fb1ad50).